lib/quarantine/databases/base.rb

```diff
@@ -8,16 +8,7 @@ class Base
 
       abstract!
 
-      Item = T.type_alias do
-        {
-          'id' => String,
-          'last_status' => String,
-          'consecutive_passes' => Integer,
-          'full_description' => String,
-          'location' => String,
-          'extra_attributes' => T.untyped
-        }
-      end
+      Item = T.type_alias { T::Hash[String, T.untyped] } # TODO: must have `id` key
```

Comment:
why do we changed this to `T::Hash[String, T.untyped]`

Reply:
I realized that the `fetch_items` and `write_items` methods could (should) be used for data of other shapes (especially since they take `table_name` as a parameter). Those methods don't refer to individual hash keys besides `id`, anyway.